docs/reference/announcements/870.md

```diff
@@ -12,82 +12,20 @@ Important changes and updates for the Camunda 8.7 release are summarized below.
-- Selected endpoints will remain active, with others scheduled for removal in the 8.9 release.
-
-### Removed: Tasklist GraphQL API
-
-With the 8.7 release, the deprecated [Tasklist GraphQL API](/apis-tools/tasklist-api/tasklist-api-overview.md) will be removed from the product.
-
-<!-- :::info
-Learn more about these updates in Upcoming API Changes in Camunda 8.
-::: -->
-
-## Identity management updates <span class="badge badge--long" title="This feature affects SaaS">SaaS</span><span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
-
-The [Identity service](/self-managed/identity/what-is-identity.md) is enhanced to deliver greater flexibility, control, and security for both Self-Managed and SaaS users. These updates are part of our broader effort to streamline the platform’s architecture.
-
-### Cluster-level identity management
-
-Identity settings will be configured at the orchestration cluster level, allowing each cluster to have unique OIDC configurations. This cluster-specific setup empowers organizations to assign different identity providers (IdPs) across clusters, offering improved control over permissions and user group mappings, resulting in a more streamlined and efficient configuration experience.
-
-For SaaS customers, identity management in Camunda 8.7 remains consistent with Camunda 8.6, allowing the attachment of a single IdP per organization. However, cluster-level identity capabilities are provided for SaaS as well as Self-Managed. This means that user groups, roles, and access permissions can now be managed at the cluster level, giving SaaS customers the same granular access control as in Self-Managed environments.
-
-### Decoupling from Keycloak <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
-
-Built-in Keycloak integration in Self-Managed is removed, allowing customers to use any compatible IdP.
-
-- Keycloak remains fully supported as an external option. For cluster-level identity management it must be connected as an external OIDC provider moving forward.
-- OpenID Connect (OIDC) remains the standard for seamless integration with chosen IdPs.
-
-### Resource-based permissions
-
-Resource-level permissions are introduced for process definitions and web applications.
-
-- Admin users retain full access, but regular users must be granted specific permissions to perform actions/view resources.
-- For organizations that build custom front-ends and access Camunda via API, users with API permissions can still access process data through the V2 API.
-
-<!-- :::info
-Learn more about these updates in Introducing Enhanced Identity Management in Camunda 8.7.
-::: -->
+  - [Web Modeler deploy diagram change](#)
+  - [HELM combined ingress](#)
+- [Southwest Asia for SaaS region](#)
 
 ## Installation and deployment updates <span class="badge badge--long" title="This feature affects Self-Managed">Self-Managed</span>
 
+<!-- NEEDS UPDATE -->
```

Comment:
@hisImminence Will we ship the following with 8.7?
* Separated Ingress removed
* Manual installation

It may be easier to _see_ this in the [preview environment](https://preview.docs.camunda.cloud/pr-4890/docs/next/reference/announcements/announcements-870/#installation-and-deployment-updates-self-managed).

Reply:
Hi @akeller, 

> Separated Ingress removed
no, this will be moved to 8.8

> Manual installation
Still unclear. I will report back once we have clarity.

Reply:
Removed the Ingress topic, and added a comment on the manual installation topic.

Reply:
I will remove! TY!